supervisor/docker/addon.py

```diff
@@ -515,7 +515,11 @@ def _run(self) -> None:
             )
 
     def _install(
-        self, version: AwesomeVersion, image: str | None = None, latest: bool = False
+        self,
+        version: AwesomeVersion,
+        image: str | None = None,
+        latest: bool = False,
+        arch: str | None = None,
```

Comment:
Let's create an Enum for Arch. See https://github.com/home-assistant/supervisor/blob/main/supervisor/const.py#L392 and then we can use it on all our Schema (see AddonStage is used). We currently converting all kinda flags into real data types